collector/collector_test.go

```diff
@@ -117,7 +117,7 @@ func TestRun(t *testing.T) {
 		Type:    tcp.EchoRequest,
 		SrcAddr: source,
 		DstAddr: target,
-		Af:      network.Family(&target),
+		Af:      defines.AfInet,
```

Comment:
This was previous code. We can address changing `defines` to `d` throughout the entire codebase in a separate diff.